common.js

```diff
@@ -188,6 +187,34 @@ module.exports = {
     }
   },
 
+  kebabProperties: {
+    'electron-version': 'electronVersion',
+    'app-copyright': 'appCopyright',
+    'app-version': 'appVersion',
+    'build-version': 'buildVersion',
+    'app-bundle-id': 'appBundleId',
+    'app-category-type': 'appCategoryType',
+    'extend-info': 'extendInfo',
+    'extra-resource': 'extraResource',
+    'helper-bundle-id': 'helperBundleId',
+    'osx-sign': 'osxSign',
+    'protocol-name': 'protocolName',
+    'version-string': 'versionString'
+  },
+
+  camelCase: function camelCase (properties, warn) {
+    Object.keys(module.exports.kebabProperties).forEach(function (key) {
+      var value = module.exports.kebabProperties[key]
+      if (properties.hasOwnProperty(key)) {
+        warn && warning(`The ${key} parameter is deprecated when used from JS, use ${value} instead`)
```

Comment:
Also, it should be mentioned that the old property is going to be removed in the next major version.